MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,569 @@
+
+For example, you can set the header toolbar to a vertical orientation:
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+var headerToolbar = imageEditor.Toolbars[0];
+headerToolbar.Orientaion = ToolbarOrientation.Vertical;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Position
+
+To adjust the alignment of the toolbar by using the `Position` property in the `ImageEditorToolbar` class. The default value is `ToolbarPosition.Start`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+var footerToolbar = imageEditor.Toolbars[1];
+footerToolbar.Position = ToolbarPosition.Start;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+### Toolbar size customization
+
+The size of the toolbar items will be adjusted according to the specified dimension by using `Size`property in the `ImageEditorToolbar` class.
+
+For horizontal orientation, the size will be considered as width, and for vertical orientation, it will be considered as height.
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image4.png");
+    imageEditor.Toolbars[0].Size = 80;
+    imageEditor.Toolbars[1].Size = 50;
+
+{% endhighlight %}
+
+## Customize default crop option
+
+Customize a crop types using `CropTypes` property in `ImageEditorToolbarSettings` class.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings CropTypes="Circle,Square,Free" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.CropTypes = ImageCropType.Square;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize default Effects option
+
+Customize a effect types using `EffectTypes` property in `ImageEditorToolbarSettings` class.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings EffectTypes="Hue, Blur" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.EffectTypes = ImageEffect.Blur;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize default shape option
+
+Customize a shape types using `EffectTypes` property in `ImageEditorToolbarSettings` class.
```

Comment:
Customize a shape types using `Shapes` property in `ImageEditorToolbarSettings` class.